centreon/bin/centreon-plugin

```diff
@@ -303,17 +299,19 @@ def applyRudderMonitoringConfigurations(conf):
         print("[!] Unable to connect to Centreon webservice. Check centreon.conf ?")
         sys.exit(1)
 
+    # for debug only first-half of synchronize-hosts
     if(args['pull']):
```

Comment:
I think you can remove "pull push and update" since theu are not documented anymore